README.md

```diff
@@ -38,9 +38,7 @@ You will need the following things properly installed on your computer.
 
 * [Git][git]
 * [Node.js][node]
-  * Supported versions:`v12`, `v14` and `v16`.
```

Comment:
Removing the notes about Apple silicon. These problems does not exist anymore, so no need to troubleshoot them.